.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -16,9 +16,6 @@ A clear and concise description of what you expected to happen.
 
 **Desktop (please complete the following information):**
 
-- OS: [e.g. iOS]
-- Browser [e.g. Chrome 84, safari 14.0]
-- Version [e.g. 22]
```

Comment:
Sorry, I wasn't clear. The `Version` line can be removed since it is redundant with the above. OS and Browser are useful. Thanks again